src/Widgets/calendar/ControlHeader.vala

```diff
@@ -19,35 +19,85 @@
 
 
 namespace DateTime.Widgets {
-    public class ControlHeader : Gtk.Box {
+    public class ControlHeader : Gtk.Grid {
         public signal void left_clicked ();
         public signal void right_clicked ();
         public signal void center_clicked ();
+
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
+        }
+
+        construct {
+            string style = """
+                .header-label {
+                    font-family: "Raleway", sans;
+                    font-weight: 300;
+                    font-size: 1.66em;
+                }
+            """;
+
+            var provider = new Gtk.CssProvider ();
+            try {
+                provider.load_from_data (style, style.length);
+                Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```

Comment:
Styles should be applied directly to the widgets, not to the whole screen. Especially because in the panel we don't want styles meant for this indicator to leak into the other indicators